html/browsers/the-window-object/proxy-getOwnPropertyDescriptor.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>getOwnPropertyDescriptor() is correct for Proxy with host object target</title>
+  <link rel="help" href="https://html.spec.whatwg.org/multipage/#window">
+  <link rel="help" href="https://heycam.github.io/webidl/#Unforgeable">
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<script>
+'use strict';
+
+test(() => {
+  let windowProxy = new Proxy(window, {});
+  name = 'old_name';
+  let descriptor = Object.getOwnPropertyDescriptor(windowProxy, 'name');
+
+  assert_equals(descriptor.get.call(window), 'old_name');
+  descriptor.set.call(window, 'new_name');
+  assert_equals(name, 'new_name');
+  assert_true(descriptor.enumerable);
+  assert_true(descriptor.configurable);
+}, 'Window target, no trap, "name" attribute');
+
+test(() => {
+  let windowProxy = new Proxy(window, {});
+
+  assert_object_equals(
```

Comment:
Please do not use this as per #2033.

Reply:
31db52f and e41d68a remove `assert_object_equals`; 432b13b adds deprecation comment in `testharness.js`